lib/experimental/block-editor-settings-mobile.php

```diff
@@ -20,10 +20,15 @@ function gutenberg_get_block_editor_settings_mobile( $settings ) {
 		defined( 'REST_REQUEST' ) &&
 		REST_REQUEST &&
 		isset( $_GET['context'] ) &&
-		'mobile' === $_GET['context'] &&
-		WP_Theme_JSON_Resolver_Gutenberg::theme_has_support()
+		'mobile' === $_GET['context']
 	) {
-		$settings['__experimentalStyles'] = gutenberg_get_global_styles();
+		if ( WP_Theme_JSON_Resolver_Gutenberg::theme_has_support() ) {
+			$settings['__experimentalStyles'] = gutenberg_get_global_styles();
+		}
+
+		// To be set to true when the web makes quote v2 (inner blocks) the default.
+		// See https://github.com/WordPress/gutenberg/pull/25892.
+		$settings['__experimentalEnableQuoteBlockV2'] = false;
```

Comment:
While we don't make the quote v2 the default, the user can still activate it for testing if they go to "Gutenberg > Experiments". @geriux would you want / does it make sense that this reads that value as well? It can perhaps help testing.

Reply:
I think that would be a good thing to have. Would it require too many changes or is it easy to get that value here as well?